### PR TITLE
MINOR: Add Jenkinsfile, used common parent, and added licenses

### PR DESCRIPTION
**Pull request by rhauch**
Included licenses for all of the dependencies, and added Jenkinsfile for builds.

### CODE REVIEW

File: licenses/LICENSE-kafka-connect-datagen-0.1.1-SNAPSHOT.txt [hunk 1]
```diff
@@ -0,0 +1,422 @@
+<!DOCTYPE html>
```
Comment (wicknicks) [inline licenses/LICENSE-kafka-connect-datagen-0.1.1-SNAPSHOT.txt]: do we need this file?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env groovy
+/*
+ * Copyright [2018 - 2018] Confluent Inc.
+ */
+common {
+  slackChannel = '#connect-eng'
+  nodeLabel = 'docker-oraclejdk8'
```
Comment (wicknicks) [inline Jenkinsfile]: PS: we can use docker10 if the default docker11 is causing issues (ref: https://github.com/confluentinc/connect-plugins-common/blob/master/Jenkinsfile#L7).

File: licenses/LICENSE-kafka-connect-datagen-0.1.1-SNAPSHOT.txt (hunk 1 above)
Comment (rhauch) [reply licenses/LICENSE-kafka-connect-datagen-0.1.1-SNAPSHOT.txt]: This is what the license tool generates.

File: Jenkinsfile (hunk 2 above)
Comment (rhauch) [reply Jenkinsfile]: My concern with using JDK 10 or 11 is that we don't know whether the connector actually works with JDK 8. I think it's better to build connectors with older JDKs and older APIs. Ideally, we'd be able to run against multiple JDKs, but not sure how to configure that.

**Comment by rhauch** [comment]
@ybyzek, yes, the next release will be 0.1.1.

**Comment by rhauch** [comment]
@ybyzek, I will update the README, though.

**Comment by ybyzek** [comment]
@rhauch , if next release is 0.1.1, we have to update also:

- Dockerfile-confluenthub
- Dockerfile-local
- README.md
- docker-compose.yml

**Comment by rhauch** [comment]
@ybyzek, I've updated the README and Docker-related files.

**Pull request merged**